<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapCounterClass</class>
 <widget class="QDialog" name="SnapCounterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>305</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Counter_SnapCounter</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>0</y>
     <width>541</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/SnapCounter/Resources/Background.bmp)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>48</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>241</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>311</width>
      <height>241</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtCounterValue">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>171</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="CounterValueList">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>46</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>171</width>
       <height>181</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>66</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfig">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>184</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Configure...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>48</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>201</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>6</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>Snap source:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listSnapSource">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>152</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Time interval:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTimeInterval">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>148</y>
       <width>73</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>73</width>
       <height>25</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>179</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>(From 0.02Hz to 50kHz)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>259</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string>Snap count:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSnapCount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>259</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>25</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="snapcounter.qrc"/>
 </resources>
 <connections/>
</ui>
